templates/named.conf.options.erb

```diff
@@ -22,9 +22,9 @@ options {
 <% end -%>
 
 <% if @listen_on.size == 0 then -%>
```

Comment:
Nope, the port can only be specified in the `listen-on` statement.  See here: http://docstore.mik.ua/orelly/networking_2ndEd/dns/ch10_15.htm

My other thought was to allow a hash for the `$listen_on` argument, like so:

```
$listen_on = {
'10.0.0.10' => 5300
}
```
..since in BIND 9 you can actually set the port on a per IP basis, but there'd still need to be a `$listen_on_port` option for when no `$listen_on` addresses are defined (i.e. `any`).